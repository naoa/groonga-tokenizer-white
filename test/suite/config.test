plugin_register tokenizers/white

config_set tokenizer-white.table white_terms

table_create white_terms TABLE_PAT_KEY ShortText
load --table white_terms
[
{"_key": "処理"}
]

tokenize TokenWhite "情報処理装置"
