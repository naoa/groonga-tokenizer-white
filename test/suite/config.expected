plugin_register tokenizers/white
[[0,0.0,0.0],true]
config_set tokenizer-white.table white_terms
[[0,0.0,0.0],true]
table_create white_terms TABLE_PAT_KEY ShortText
[[0,0.0,0.0],true]
load --table white_terms
[
{"_key": "処理"}
]
[[0,0.0,0.0],1]
tokenize TokenWhite "情報処理装置"
[[0,0.0,0.0],[{"value":"処理","position":0,"force_prefix":false}]]
